docs/topics/launcher-review-update-booster-manual.adoc

```diff
@@ -2,7 +2,7 @@
 
 .Next Steps: Update your Booster Manually
 
-You can build and deploy your booster application using the Fabric8 Maven Plugin to launch the S2I process on OpenShift.
+You can build and deploy your booster application using the Fabric8 Maven Plugin to launch the link:{link-s2i-process}[S2I process] on OpenShift.
```

Comment:
This sentence should be updated according to my previous remark - correct ?

Reply:
see prior comment.